server/webapp/WEB-INF/urlrewrite.xml

```diff
@@ -136,12 +136,19 @@
 
     <rule>
         <name>Rails API</name>
-        <from>^/(api/(?!admin/config).*)$</from>
+        <from>^/(api/((?!admin/config)(?!users/)).*)$</from>
```

Comment:
@maheshp - can we pull this `/users` into a separate rule — while this would be a bit noisy, IMO it'll help when removing individual paths from the rewrite.

WDYT?